scripts/alpha-v5.mjs

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env zx
 
-import 'zx/globals';
+import { $, echo } from 'zx';
 
 import { constants } from './common.mjs';
 
 await $`npx json -I -f ${constants.ChangesetConfigFile} -e "this.changelog = false"`;
 
-const res = await $`npx changeset version --snapshot canary`;
+const res = await $`npx changeset version --snapshot alpha-v5`;
```

Comment:
Snapshot is the wrong thing to do here as it won't publish at the next major, but the current version + the tag. See https://github.com/changesets/changesets/blob/main/docs/dictionary.md:

> snapshot prerelease - A snapshot pre-release is intended to make it easy to test all changes at a particular git commit. It is published at 0.0.0 and uses the git hash as the tag. This should be used as a less formal method than a release candidate to make it easy to install and test changes. An example, if we have a package at 1.0.0, and have a changeset for it to be major bumped, a snapshot release will be at 0.0.0-ABCDEFGH (the last github hash). If we add more commits, the next snapshot would be 0.0.0-HIJKLMNOP.

We'll want to use pre-release mode as explained in the same doc and by Andarist: https://github.com/changesets/action/issues/69#issuecomment-774909280

We can probably look into how stately did it: https://github.com/statelyai/xstate/pull/902

Reply:
Stately did it like this: https://github.com/statelyai/xstate/pull/878

Reply:
@LekoArts check the `To verify alpha-v5 tags` steps of the descriptions and check if the packages are published under the next major version.
This may not the best way to make the pre-release tag of the next major version but since we have this mechanism i would go with this one.